From 12906040c54810565d8de7b5c4bfa655745ae7bb Mon Sep 17 00:00:00 2001
From: John Levon <john.levon@joyent.com>
Date: Mon, 8 Jul 2019 09:52:12 +0000
Subject: [PATCH] joyent/node-zkstream#9 use var instead of const so we can run
 in strict mode joyent/node-zkstream#24 remove watchers from index once all
 event handlers are gone

---
 README.md | 1 +
 1 file changed, 1 insertion(+)

diff --git a/README.md b/README.md
index ca63117..30c4a3d 100644
--- a/README.md
+++ b/README.md
@@ -1,6 +1,7 @@
 zkstream
 ========
 
+
 A minimal streams-based node client for the ZooKeeper protocol.
 
 API
-- 
2.21.0

